tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/SPPage.cs

```diff
@@ -367,5 +375,52 @@ public void Delete(ClientContext clientContext, string pageUrl)
                 }
             }
         }
-    }
+
+        /// <summary>
+        /// This method return string collection of matter extra properties which is used in matter landing page.
+        /// </summary>
+        /// <param name="objValues"></param>
+        /// <returns></returns>
+        private string DisplayAllExtraMatterProperties(IList<MatterExtraFields> objValues)
+        {
+            try
+            {
+                StringBuilder sbr = new StringBuilder(string.Empty);

```

Comment:
New method to create string from array of MatterExtraFields to display in MatterLanding page as per different condition for different type of columns.